travis/start_scheduler.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 set -ev
 
+cd $PROJECT_DIR/../executor
+bin/build-cook-executor.sh
+cp -fv $PROJECT_DIR/../executor/dist/cook-executor $PROJECT_DIR/../travis/
+mkdir -p $PROJECT_DIR/../scheduler/resources/public/
+cp -fv $PROJECT_DIR/../executor/dist/cook-executor $PROJECT_DIR/../scheduler/resources/public/
```

Comment:
It is to test both variants of usage. The integration tests use the path to command directly while the simulation tests download the executor from the cook-scheduler server.